x-pack/plugins/threat_intelligence/cypress/integration/sources/indicators.spec.ts

```diff
@@ -13,24 +13,53 @@ import {
   FLYOUT_TITLE,
```

Comment:
Considering how much time a Kibana takes to initialize, I feel like we should minimize the number of tests we have. If for each test Kibana takes 10-20 seconds to initialize we're going to blow out CI time quickly...
To me the 3 tests in this file could be easily merged into one:
- load the page with data
- verify the table
- take a row, click on the flyout button
- verify the flyout component
- go back to the page, and visit a second page with this time no data
- verify

Thoughts?

Reply:
Kibana / ES is only initialized once, during the FTR setup phase, so it does not really matter how much tests you have. Of course, individual test cases take time and I could squash some of these cases into one thing anyway. Will do it:)